src/services/AdvancedTradingView/adapters/uniswapV3/uniswapV3.queries.ts

```diff
@@ -0,0 +1,61 @@
+import { gql } from 'graphql-request'
+
+export const UNISWAP_PAIR_SWAPS = gql`
+  query getPairTrades($token0_in: [Bytes]!, $token1_in: [Bytes]!, $first: Int!, $skip: Int!) {
+    swaps(
+      where: { token0_in: $token0_in, token1_in: $token1_in }
+      first: $first
+      skip: $skip
+      orderDirection: "desc"
+      orderBy: "timestamp"
+    ) {
+      id
+      transaction {
+        id
+      }
+      amount1
+      amount0
+      amountUSD
+      timestamp
+    }
+  }
+`
+
+export const UNISWAP_PAIR_BURNS_AND_MINTS = gql`
+  query getPairActivity($token0_in: [Bytes]!, $token1_in: [Bytes]!, $first: Int!, $skip: Int!) {
+    mints(
+      where: { token0_in: $token0_in, token1_in: $token1_in }
+      first: $first
+      skip: $skip
+      orderDirection: "desc"
+      orderBy: "timestamp"
+    ) {
+      id
+      transaction {
+        id
+      }
+      amount0
+      amount1
+      amountUSD
+      timestamp
+      type: __typename
+    }
+    burns(
+      where: { token0_in: $token0_in, token1_in: $token1_in }
+      first: $first
+      skip: $skip
+      orderDirection: "desc"
+      orderBy: "timestamp"
+    ) {
+      id
+      transaction {
+        id
+      }
+      amount0
+      amount1
+      amountUSD
+      timestamp
+      type: __typename
+    }
```

Comment:
I think I cannot use `fragment` here because I need specify `name` of entity. I need to create at least 3 fragments 

Here is a example from documentation. 
```js
fragment CoreCommentFields on Comment {
    id
    postedBy {
      username
      displayName
    }
    createdAt
    content
  }
```
 For now I use
```js
export const BASE_QUERY_FRAGMENT = 'id transaction {id} amount0 amount1 amountUSD timestamp'
```